chain/x/zoracle/internal/types/msgs.go

```diff
@@ -111,6 +177,57 @@ func (msg MsgReport) GetSignBytes() []byte {
 	return sdk.MustSortJSON(bz)
 }
 
+// MsgReportData defines a new version Report message
```

Comment:
MsgReport is a message sent by each of the block validators to respond to a data request.